MangoAPI.BusinessLogic/DependencyInjection/AzureBlobInitializer.cs

```diff
@@ -0,0 +1,45 @@
+using MangoAPI.Application.Interfaces;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.DependencyInjection;
+using System;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace MangoAPI.BusinessLogic.DependencyInjection;
+
+public static class AzureBlobInitializer
+{
+    public static async Task InitializeAzureBlobAsync(this IApplicationBuilder app)
```

Comment:
Implement this as a void, because you cannot call async API properly in startup, there always will be blocking request